docs/gambit/mutation-verifier.md

```diff
@@ -1,17 +1,16 @@
 # Using Gambit with the Prover
 
-This is the mutation verifier which
- checks that variants of the original
- solidity program do not pass the specification.
+The mutation verifier checks that variants of the original
+  Solidity program do not pass verification against a given specification.
 It uses mutations from the {doc}`Gambit <gambit>`
```

Comment:
The tool uses random mutations generated by gambit as well as (optionally) manual mutants generated specifically by the user

Reply:
good, changed